scipy/stats/_distribution_infrastructure.py

```diff
@@ -0,0 +1,5082 @@
+            method = self._entropy_quadrature
+        return method
+
+    def _entropy_formula(self, **kwargs):
+        raise NotImplementedError(self._not_implemented)
+
+    def _entropy_logexp(self, **kwargs):
+        return np.real(np.exp(self._logentropy_dispatch(**kwargs)))
+
+    def _entropy_quadrature(self, **kwargs):
+        def integrand(x, **kwargs):
+            pdf = self._pdf_dispatch(x, **kwargs)
+            return np.log(pdf)*pdf
+        return -self._quadrature(integrand, kwargs=kwargs)
+
+    @_set_invalid_nan_property
+    def median(self, *, method=None):
+        r"""Median
+
+        If a continuous random variable :math:`X` has probability :math:`0.5` of
+        taking on a value less than :math:`m`, then :math:`m` is the median.
+        That is, the median is the value :math:`m` for which:
+
+        .. math::
+
+            P(X ≤ m) = 0.5 = P(X ≥ m)
+
+        Parameters
+        ----------
+        method : {None, 'formula', 'icdf'}
+            The strategy used to evaluate the median.
+            By default (``None``), the infrastructure chooses between the
+            following options.
+
+            - ``'formula'``: use a formula for the median
+            - ``'icdf'``: evaluate the inverse CDF of 0.5
+
+            Not all `method` options are available for all distributions.
+            If the selected `method` is not available, a ``NotImplementedError``
+            will be raised.
+
+        Returns
+        -------
+        out : Array
+            The median
+
+        See Also
+        --------
+        ContinuousDistribution.mean
+        ContinuousDistribution.mode
+        ContinuousDistribution.icdf
+
+        Examples
+        --------
+        Instantiate a distribution with the desired parameters:
+
+        >>> from scipy import stats
+        >>> X = stats.Uniform(a=0., b=10.)
+
+        Evaluate the inverse CDF at the desired argument:
+
+        >>> X.median()
+        5
+        >>> X.median() == X.icdf(0.5) == X.iccdf(0.5)
+        True
+
+        """
+        return self._median_dispatch(method=method, **self._parameters)
+
+    @_dispatch
+    def _median_dispatch(self, method=None, **kwargs):
+        if self._overrides('_median_formula'):
+            method = self._median_formula
+        else:
+            method = self._median_icdf
+        return method
+
+    def _median_formula(self, **kwargs):
+        raise NotImplementedError(self._not_implemented)
+
+    def _median_icdf(self, **kwargs):
+        return self._icdf_dispatch(0.5, **kwargs)
+
+    @_set_invalid_nan_property
+    def mode(self, *, method=None):
+        r"""Mode
+
+        Informally, the mode is a value that a random variable has the highest
+        probability (density) of assuming. That is, the mode is the element of
+        the support :math:`\chi` that maximizes the probability density
+        function :math:`f(x)`:
+
+        .. math::
+
+            \text{mode} = \argmax_{\chi} f(x)
```

Comment:
\argmax_{x \in \chi}

Reply:
It is not uncommon for this to be implicit, just like the limits of integration are implicit in many integrals.

![image](https://github.com/mdhaber/scipy/assets/6570539/26d976f6-3ac6-4ac1-a968-771924fd9039)

Reply:
Done. If it were to be implicit, it would be `\argmax_x`, not `\argmax_{\chi}`.

Reply:
can be resolved